src/Movie.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+class Movie {
+    private string $name;
+    private string $director;
+    private string $genre;
+    private int $year;
+
+    public function __construct(string $name, string $director, string $genre, int $year)
+    {
+        $this->name = $name;
+        $this->director = $director;
+        $this->genre = $genre;
+        $this->year = $year;
+    }
+
+    public function createCard(): string
+        {
+            return
+            "<div class='card'>" .
```

Comment:
needs to be double quotes around card - in HTML, attributes should always use double quotes